README.md

```diff
@@ -45,8 +46,9 @@ cargo +esp build --release --features=esp32s3 --target=xtensa-esp32s3-none-elf
 In order to generate the JSON stub files for one or more devices, you can again use the `xtask` package:
```

Comment:
```suggestion
In order to generate the JSON and TOML stub files for one or more devices, you can again use the `xtask` package:
```

Reply:
Did another round of testing and everything seems to be working fine:
|   Chip   | UART | USB Serial JTAG |
| :------: | :--: | :-------------: |
|  ESP32   |  ✅  |       N/A       |
| ESP32-C2 (40MHz) |  ✅  |       N/A       |
| ESP32-C3 |  ✅  |        ✅        |
| ESP32-C6 |  ✅  |       ✅        |
| ESP32-H2 |  ✅  |       ✅        |
| ESP32-S2 |  ✅  |       N/A       |
| ESP32-S3 |  ✅  |       ✅        |